exporter/exporter.go

```diff
@@ -75,7 +75,7 @@ func Start(client rpc.Client) error {
 			logger.Fatal(err)
 		}
 
-		for epoch := head.HeadEpoch - 1; epoch >= 0; epoch-- {
+		for epoch := head.HeadEpoch - 1; true; epoch-- {
```

Comment:
true here, is that right?

Reply:
there might be a chance that epoch goes below 0 when the client returns an error and continue is called

Reply:
epoch is an uint hence can never go below 0

Reply:
https://go.dev/play/p/w3U4mMxbPCL